src/GraphbackClient.ts

```diff
@@ -54,7 +54,7 @@ export class GraphbackClient {
   }
 }
 
-export async function initGraphbackClient(
+export async function graphbackClientBuilder(
```

Comment:
Small note/suggestion: The naming for `graphbackClientBuilder` and for `graphbackServerBuilder` is a bit strange because the naming sounds like an object/class (because of the word builder) as opposed to an action/function. There is also a concept called the [builder pattern](https://en.wikipedia.org/wiki/Builder_pattern) which this could be confused with.

I think `buildGraphbackClient` or `createGraphbackClient` would have less potential to confuse somebody.

Reply:
I agree I've renamed them to `buidBla`